src/transformers/models/auto/modeling_auto.py

```diff
@@ -545,6 +545,13 @@
     ]
 )
 
+MODEL_FOR_VISUAL_QUESTION_ANSWERING_MAPPING_NAMES = OrderedDict(
+    [
+        # Model for Visual Question Answering mapping
```

Comment:
```suggestion
```
This comment is not necessary, the constant name says the same thing (other classes have the comments because of our old model templates, we don't need to duplicate that ;-) )